### PR TITLE
Update dashboard: alex

### LINKED ISSUE #251: Update dashboard to include NASA images
As an astronomer I like cool pictures, when I'm stargazing I would like to see cool pictures so integrate it into the dashboard for ease of use.

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #251 

I have updated the dashboard homepage to include nasa picture of the day and the ISS current location

### CODE REVIEW

File: dashboard/dashboard.py [hunk 1]
```diff
@@ -63,6 +63,34 @@ def validate_phone_number(phone_number:str, region='GB') -> bool:
...
+             stargazers so that anyone can enjoy the wonders of space.""")
+    nasa_pipeline()
+    image_title, image = get_image_of_the_day()
+    iss = get_iss_location()
+    col1, col2 = st.columns(2)
+
+    with col1:
+        st.header('Image of the day')
+        st.write(f'{image_title} - {datetime.now().strftime("%d/%m/%Y")}')
+        if 'youtube' in image:
```
Comment (CalebMathers) [inline dashboard/dashboard.py:77]: Could this cause issues if the video was from a different source?

File: dashboard/dashboard.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 import numpy as np
 
 from load_dashboard_data import connect_to_db, load_from_starwatch_rds, load_forecasts_by_county_name
-
+from image_pipeline import nasa_pipeline,get_image_of_the_day,get_iss_location
```
Comment (olliejth) [inline dashboard/dashboard.py:19]: Could be spaces after commas for appearance.

File: dashboard/image_pipeline.py [hunk 3]
```diff
@@ -15,6 +15,16 @@
 
 KEYS = ['data','title','url']
 
+def get_image_of_the_day() -> str:
```
Comment (CalebMathers) [inline dashboard/image_pipeline.py:18]: What a descriptive function name, incredible

File: dashboard/dashboard.py [hunk 4]
```diff
@@ -16,7 +16,7 @@
...
 
 load_dotenv()
 st. set_page_config(layout="wide")
```
Comment (olliejth) [inline dashboard/dashboard.py:22]: I like this better that how it was before.

File: dashboard/dashboard.py [hunk 5]
```diff
@@ -63,6 +63,34 @@ def validate_phone_number(phone_number:str, region='GB') -> bool:
...
+        st.write(f'Last updated: {iss["timestamp"].strftime("%H:%M:%S %d/%m/%Y")}')
+        iss_df = pd.DataFrame({
+            'latitude': [float(iss['latitude'])],
+            'longitude': [float(iss['longitude'])]
+        })
+        zoom_level = 1.5
+        st.map(iss_df, zoom=zoom_level)
+
+
+
```
Comment (olliejth) [inline dashboard/dashboard.py:93]: Descriptive naming.

File: dashboard/dashboard.py [hunk 6]
```diff
@@ -63,6 +63,34 @@ def validate_phone_number(phone_number:str, region='GB') -> bool:
...
+            st.image(image)
+
+    with col2:
+        st.header('Current ISS Location')
+        st.write(f'Last updated: {iss["timestamp"].strftime("%H:%M:%S %d/%m/%Y")}')
+        iss_df = pd.DataFrame({
+            'latitude': [float(iss['latitude'])],
+            'longitude': [float(iss['longitude'])]
+        })
+        zoom_level = 1.5
```
Comment (CalebMathers) [inline dashboard/dashboard.py:89]: This is my favourite zoom level

**Review by CalebMathers — APPROVED** [review]

File: dashboard/image_pipeline.py [hunk 7]
```diff
@@ -15,6 +15,16 @@
...
+    query = """SELECT image_name,image_url FROM image
+    WHERE image_name ILIKE '%nasa_%' AND
+     image_date = CURRENT_DATE """
+    with get_connection() as conn:
+        with conn.cursor() as cur:
+            cur.execute(query)
+            data = cur.fetchone()
+            print(data)
```
Comment (olliejth) [inline dashboard/image_pipeline.py:26]: Nice use of with.

File: dashboard/image_pipeline.py [hunk 8]
```diff
@@ -93,3 +103,4 @@ def get_iss_location() -> dict:
     logger = logging.getLogger(__name__)
     logging.basicConfig(level=logging.INFO)
     load_dotenv()
+    get_image_of_the_day()
```
Comment (olliejth) [inline dashboard/image_pipeline.py:106]: Good idea to implement this.

**Review by olliejth — APPROVED** [review]
Good clean code.

The inclusion of is_nasa_photo was a nice addition.

**Pull request merged**